### PR TITLE
David

### PR DESCRIPTION
**Pull request by duendwang**
Created articles for hosts file fix and troubleshooting login issues.

### CODE REVIEW

**Comment by duendwang** [comment]
Did I do this correctly?

Also I just remembered that the mac and Linux screenshots are missing for the hosts file fix page. Let me know if you want to merge first and then PR again with the screenshots, or wait until those screenshots are in.

**Comment by duendwang** [comment]
Added linux and mac screenshots to hosts file guide.
Redid numbering formatting in tables to be cleaner and converted markdown formatting to HTML inside tables.

File: _help/cannot-log-in/index.markdown [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+## Can Log in on Another Device, both on Same Network and on Data
+
+The issue is with your computer. There is either something running on your computer that is blocking Minecraft logins or some setting that's preventing Minecraft from reaching authentication servers. See the [list of known incompatible software](/_help/known-incompatible-software/).
+
+One of the most common causes comes from the use of illegal account generators. These programs change settings on the computer that prevents Minecraft from logging in normally, even after the account generator program is removed. To undo the changes done by these account generators, see [this article](/_help/fix-hosts-file).
+
+## Further Help
+
+If you are still not able to log in, but can log in on other devices. You can join the [Minecraft Support Discord](https://discord.gg/58Sxm23) for further help and troubleshooting. As with any online community, be sure to read the rules upon joining, and follow them.
```
Comment (LordRalex) [inline _help/cannot-log-in/index.markdown:39]: I think we have a page for technical resourrces, would prefer that page to be used.

File: _help/hosts-file/index.markdown [hunk 2]
```diff
@@ -0,0 +1,237 @@
...
+<table>
+  <tr>
+    <td>
+      <ul>
+        <li>"The authentication servers are down for maintenance."</li>
+        <li>"Please switch to Mojang mode."</li>
+        <li>"Not authenticated with minecraft.net."</li>
+      </ul>
+    </td>
+    <td><img src="https://cdn.discordapp.com/attachments/607129205981052929/801902607203172392/unknown.png" height=200></td>
```
Comment (LordRalex) [inline _help/hosts-file/index.markdown:25]: Upload the images to the repo instead of using it from Discord

**Review by LordRalex — CHANGES_REQUESTED** [review]

**Review by LordRalex — APPROVED** [review]

**Pull request merged**